infra/gcp/terraform/kubernetes-public/k8s-prod-readiness.tf

```
@@ -0,0 +1,51 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+ 
+/*
+This file defines:
+- A BigQuery dataset for PRR reporting
+- IAM bindings on that dataset
+*/
+
+locals {
+  prod-readiness_owners = "k8s-infra-prr@kubernetes.io"
```

```suggestion
  prod-readiness_owners = "k8s-infra-prod-readiness@kubernetes.io"
```

OR update the OWNERS file.